rs/sns/integration_tests/src/proposals.rs

```diff
@@ -1759,7 +1760,7 @@ fn test_change_voting_rewards_round_duration() {
         delta_s += current_voting_rewards_round_duration_seconds as i64;
         sns_canisters.set_time_warp(delta_s).await.unwrap();
         let reward_event_1 = sns_canisters
-            .await_reward_event_after(reward_event_0.end_timestamp_seconds.unwrap())
+            .await_reward_event_after(reward_event_0.end_timestamp_seconds.unwrap(), &runtime)
```

Comment:
I don't think you should need &runtime here, b/c sns_canisters keeps a reference to runtime?  Or the canisters in it do?

Reply:
Also, you could just directly access the state_machine with a match statement here, or a function that just takes the runtime directly, instead of complicating the interface here?  the test gets the runtime as a parameter.

Reply:
Done!